src/BuildScriptGenerator/Python/PythonPlatform.cs

```diff
@@ -144,6 +161,26 @@ public PlatformDetectorResult Detect(RepositoryContext context)
             var packageDir = GetPackageDirectory(context);
             var virtualEnvName = GetVirtualEnvironmentName(context);
             var isPythonPackageCommandEnabled = _commonOptions.ShouldPackage;
+            var pythonPackageWheelType = GetPythonPackageWheelType(context);
+
+            if (!isPythonPackageCommandEnabled && !string.IsNullOrWhiteSpace(pythonPackageWheelType))
+            {
+                throw new InvalidUsageException($"Option '{PythonPackageWheelPropertyKey}' cann't exist" +
+                    $"without package command being enabled. Please provide --package along with wheel type");
+            }
+
+            if (isPythonPackageCommandEnabled &&
+                !string.IsNullOrWhiteSpace(pythonPackageWheelType) &&
+                !string.Equals(pythonPackageWheelType.ToLower(), "universal"))
+            {
+                throw new InvalidUsageException($"Option '{PythonPackageWheelPropertyKey}' can only have 'universal' as value.");
+            }
+            else if (isPythonPackageCommandEnabled &&
+                !string.IsNullOrWhiteSpace(pythonPackageWheelType) &&
+                string.Equals(pythonPackageWheelType.ToLower(), "universal"))
```

Comment:
nit: I would consolidate this `else if` with the `if` above, just to clean it up a little:

```cs
if (isPythonPackageCommandEnabled && !string.IsNullOrEmptyWhiteSpace(pythonPackageWheelType))
{
    if (!string.Equals(pythonPackageWheelType.ToLower(), "universal")
    {
        throw new InvalidUsageException($"Option '{PythonPackageWheelPropertyKey}' can only have 'universal' as value.'");
    }
    
    manifestFileProperties[PythonManifestFilePropertyKeys.PackageWheel] = pythonPackageWheelType;
}
```

Also, the logic for the nested `if` will need to change if you do decide to go with the array of supported types 😄

Reply:
same comment about arraymodel